04-onion-routing.md

```diff
@@ -470,6 +470,15 @@ func NewOnionPacket(paymentPath []*btcec.PublicKey, sessionKey *btcec.PrivateKey
 	var mixHeader [routingInfoSize]byte
 	var nextHmac [hmacSize]byte
 
+        // Our starting packet needs to be padded with random bytes, so we read
```

Comment:
indentation for these first three lines is off

Reply:
Looked OK in vim for me...attempt a re-fold.